app/components/containerConsole/containerConsole.html

```diff
@@ -41,7 +41,7 @@
             <div class="form-group">
               <label for="username" class="col-lg-1 text-left col-sm-2 control-label">
                 User
-                <portainer-tooltip position="bottom" message="Format is one of: user, user:group, uid or uid:gid"></portainer-tooltip>
+                <portainer-tooltip icon="question-circle" position="bottom" message="Format is one of: user, user:group, uid or uid:gid"></portainer-tooltip>
```

Comment:
Why did you add an icon property to the `portainer-tooltip` ?

Reply:
To use the same directive with the new icons in the images view, which show the list of containers or children in a tooltip. This is a "preview" to avoid jumping to the image details view if only the names are to be checked.